### ISSUE TITLE
Question About DataSource Config Settings

### ISSUE THREAD

**Issue by jeffscottbrown** [micronaut-projects/micronaut-starter#1502]
At https://micronaut.io/launch?type=DEFAULT&name=demo&package=com.example&javaVersion=JDK_11&lang=JAVA&build=GRADLE&test=JUNIT&features=data-jpa&features=hibernate-jpa&version=3.7.2&activity=preview&showing=/src/main/resources/application.yml is the following:

```
micronaut:
  application:
    name: demo
datasources:
  default:
    url: jdbc:h2:mem:devDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
    username: sa
    password: ''
    driverClassName: org.h2.Driver
    schema-generate: CREATE_DROP
    dialect: H2
jpa.default.properties.hibernate.hbm2ddl.auto: update
netty:
  default:
    allocator:
      max-order: 3
```

Notice that `datasources.default.schema-generate` is `CREATE_DROP` and `jpa.default.properties.hibernate.hbm2ddl.auto` is `update`.

I have not confirmed, but it appears that `jpa.default.properties.hibernate.hbm2ddl.auto` is respected and `datasources.default.schema-generate` may be ignored.

I asked if `datasources.default.schema-generate` should have an impact from that config and @sdelamo responded with the following:

> Probably it should be both the same. Can you create an issue?

**Comment by dstepanov** [comment]
`schema-generate` is a Micronaut Data property that can use an internal (more limited than Hibernate) generator to create the schema. The preferred way would be to use the Hibernate schema generation in this case.

**Comment by jeffscottbrown** [comment]
> schema-generate is a Micronaut Data property that can use an internal (more limited than Hibernate) generator to create the schema. The preferred way would be to use the Hibernate schema generation in this case.

Thank you.

The question I am asking is in the config shown above, is there an expectation that the value of `datasources.default.schema-generate` has an impact on the system?

Thank you again!

**Comment by dstepanov** [comment]
It’s going to run but it might be overridden/regenerated by the Hibernate.

**Comment by jeffscottbrown** [comment]
> It’s going to run but it might be overridden/regenerated by the Hibernate.

Do you think that the `schema-generate` one should be omitted from the default generated config for a case like this?  

I think having them both there in the default generated config file for these features but having it not be used will cause confusion.

Thanks for your feedback!

**Comment by dstepanov** [comment]
Yea, it’s not needed for the hibernate

**Comment by jeffscottbrown** [comment]
FYI... I don't think this is still an issue in recent versions of Micronaut.

**Comment by sdelamo** [comment]
@wetted  change it so that it `datasources.default. schema-generate` is generated with `[NONE](https://github.com/micronaut-projects/micronaut-data/blob/master/data-runtime/src/main/java/io/micronaut/data/runtime/config/SchemaGenerate.java#L37)` when hibernate jpa is used.

**Comment by wetted** [comment]
> FYI... I don't think this is still an issue in recent versions of Micronaut.

Correct. This isn't an issue anymore. It was changed for Micronaut 3.8.0 on #1537, which in turn addressed #1524 and #1525. As a result we no longer even set the `datasources.default.schema-generate` property.

Closing since there is nothing to do as far as I can tell.
